source/_docs/configuration/templating.markdown

```diff
@@ -480,4 +480,4 @@ The default priority of operators is that the filter (`|`) has priority over eve
 ```
 {% endraw %}
 
-Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by that.
+Would round `states('sensor.temperature')` to 2 decimal places, then divide `10` by that.
```

Comment:
Actually, that is incorrect as well. 

As the filter operator `|` has priority, the `10` is rounded to 2 decimal places and the state value of `sensor.temperature` is casted/converted to a float type. Then the division takes place.

```suggestion
Would convert the state value of `sensor.temperatureround` to a float type, rounds `10` to 2 decimal places. As the final step, the float converted sensor value is divided by `10` (rounded to 2 decimals).
```

Reply:
Hello Frenck,  you are probably right but it is now beyond my understanding, the code example provided rounds 10 (the number?) to 2 decimal ? is there an interest to do so ? I'm a bit puzzled sorry, I though I had understood the example but now it makes no sense to me ^^

Reply:
Well, I guess the thing this example tries to show, is that something is happening one would not expect: The `10` gets rounded.

All `|` operators are running before anything else first. Meaning the `10` gets rounded.